sympy/printing/str.py

```diff
@@ -44,6 +44,16 @@ def emptyPrinter(self, expr):
         else:
             return str(expr)
 
+    def _print_Reals(self,expr):
+        return "S.Reals"
+
+    def _print_Complexes(self,expr):
+        return "S.Complexes"
+
+    def _print_Naturals(self,expr):
```

Comment:
Thanks for the review. I will keep the alphabetical order in mind. Also for pretty printing the intervals, would shifting the function that prints intervals to pprint.py work?